packages/@glimmer/manager/lib/internal/defaults.ts

```diff
@@ -0,0 +1,50 @@
+import { buildCapabilities } from '../util/capabilities';
+
+import type { CapturedArguments as Arguments, HelperCapabilities } from '@glimmer/interfaces';
+
+type FnArgs<Args extends Arguments = Arguments> =
+  | [...Args['positional'], Args['named']]
+  | [...Args['positional'], {}];
```

Comment:
I think this is wrong now (if we decided to go with not always passing the `{}`). At this point, isn't it just `type AnyFunction =  (...unknown[]): unknown`?

Reply:
yup. updates incoming :partying_face:

Reply:
fixed